### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by abreham1222**
In this PR this action is performed:-
1. Added index.html file for mobile menu
2. Added styles for the mobile menu
3. Added js file for mobile menu using DOM manipulation.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const humburger = document.querySelector('.humburger');
+const navMenu = document.querySelector('.navbar-desk');
+
```
Comment (tufoinnkuo10) [inline script.js:3]: - [x] Great job. everything looks ok. lets perfect it by shifting the close button inside a little so it more visible
 - [x] please shift the menu links down a little and make their font weight bolder . great job
 yours
Screenshot (418)_LI
figma

Screenshot (419)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @abreham1222 

Great job so far. You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review
- [ Required] Please you are missing one of the linter checks (Eslint). please kindly add it as it helps check your Javascript code. you can see how to add it [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: script.js (hunk 1 above)
Comment (abreham1222) [reply script.js:3]: Hi @tufoinnkuo10 
I added the eslintrc files but it is not checked anything yet kindly can you help me with this problem?

File: style.css [hunk 2]
```diff
@@ -27,9 +27,9 @@ nav {
   display: block;
   width: 25px;
   height: 3px;
-  margin: 5px 20px;
+  margin: 5px 30px;
```
Comment (BenMukebo) [inline style.css:30]: - [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇

**Review by BenMukebo — APPROVED** [review]
### Hi @abreham1222, @Shedrack-Sunday, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository have been set up correctly. ✔️  
-  When the user clicks the hamburger button, the mobile menu appears.✔️ 
- When the user clicks the close (X) button, the mobile menu disappears.✔️ 
- When the user clicks any of the mobile menu options, a correct part is displayed and the mobile menu disappears 👏🏻 
- You have followed  JavaScript best practices. ✔️ 


## Optional suggestions

- [OPTIONAL] Kindly add a meaningful `project title` and `project description` to your README file and remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Think of it as an elevator pitch that focuses on what and why without delving into how.

- [OPTiONAL] This is a group project. Please let's include all the authors in the author's section of the README file. This makes the README file professional and descriptive. 👨🏻‍🎨




_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**